docs/developers/dapp-developer/native-bindings.md

```diff
@@ -224,12 +228,18 @@ func main() {
 	if err != nil {
 		log.Fatalf("Failed to connect to the Ethereum client: %v", err)
 	}
-	auth, err := bind.NewTransactor(strings.NewReader(key), "<<strong_password>>")
+
+	keyDataBytes, err := os.ReadFile(keyFile)
```

Comment:
If you change the example to use a keyFile instead of hardcoding the jsonObject, you need to define it somewhere.
Probably easiest to just do ```const keyFile = `<<path to a keystore>>```

Reply:
Good point @MariusVanDerWijden 
Thinking it may be just as good to say 
```
keyDataBytes, err := os.ReadFile(<<path/to/keystore/file>>)
```

which is similar enough to what was being done previously with "<<strong password>>"

Sound good?